.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Request for new functionality
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+
+A clear and concise description of what a solution could look like.
+
+**Describe alternatives you've considered**
+
+A clear and concise description of any alternatives you've considered.
+
```

Comment:
I think it's worth thinking about marking this section as "optional". IMO this should be communicated like "dear user, if you have thought about several options to approach this problem, we'd love to hear about it". However, personally I wouldn't necessarily expect users to do this. Also weighing options against one another typically requires some amount of understanding of the existing implementation and general constraints, which I also wouldn't expect a user to have. 
However, I'm not sure how other projects handle this, so if this is common practice, feel free to disregard my comment.

**TL;DR:** I'd personally mark this as "optional" to keep the threshold for feature requests low.